documentation/SA1305.md

```diff
@@ -29,9 +29,17 @@ In addition, modern code editors such as Visual Studio make it easy to identify
 
 StyleCop assumes that any variable name that begins with one or two lower-case letters followed by an upper-case letter is making use of Hungarian notation, and will flag a violation of this rule in each case. It is possible to declare certain prefixes as legal, in which case they will be ignored. For example, a variable named *onExecute* will appear to StyleCop to be using Hungarian notation, when in reality it is not. Thus, the *on* prefix should be flagged as an allowed prefix.
 
-To configure the list of allowed prefixes, bring up the StyleCop settings for a project, and navigate to the Hungarian tab, as shown below:
-
-![](Images/HungarianSettings.JPG)
+To configure the list of allowed prefixes, use *stylecop.json* like the following:
```

Comment:
Changed file name to **bold** to match [Configuration.md](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/Configuration.md).

```suggestion
To configure the list of allowed prefixes, use **stylecop.json** like the following:
```